player/src/HttpMp4Video.tsx

```diff
@@ -95,6 +95,67 @@ export const HttpMp4Video: React.FC<HttpMp4VideoProps> = ({
 
   const __sensorTmRef = useRef<TransformationMatrix>()
 
+  /**
+   * Show debug logs with information received from
+   * 'progress' & 'timeupdate' events including the current
+   * up time, delay and end time of last buffer.
+   */
+  useEffect(() => {
+    const debugKey = localStorage.getItem('debug')
+    if (
+      // Don't initiate event listeners if debug is not on
+      debugKey !== 'msp:http-mp4-video' ||
+      videoRef.current === null ||
+      !playing
+    ) {
+      return
+    }
+
+    const onprogress = (event: Event) => {
+      const videoEl = event.currentTarget as HTMLVideoElement | null
+      if (videoEl === null) {
+        return
+      }
+
+      try {
+        const bufferedEnd = videoEl.buffered.end(videoEl.buffered.length - 1)
+        debugLog('%o', {
+          bufferedEnd,
```

Comment:
How is someone supposed to use this debug information?

I'd like to see something more concrete. Either in the debug log, as a comment here or in the documentation about how to interpret all the numbers and stuff that will come flying when they turn on debug.

Reply:
Done